windows/README.md

```diff
@@ -1,24 +1,9 @@
 # react-native-device-info Windows Implementation
 
-## Module Installation
-You can either use autolinking on react-native-windows 0.63 and later or manually link the module on earlier realeases.
+### Automatic install with autolinking
```

Comment:
What do you think about putting a note in here, for future people that are perhaps on old tools but don't read release notes? This is just the content of the release notes for the change really

```suggestion
## Minimum supported versions

- react-native-windows 0.63 or newer
- MSVC build tools v142 (included in Visual Studio 2019) or newer
- x86, x64, or arm64 are supported, arm (32-bit) is not supported

## Automatic install with autolinking
```

Reply:
That looks good to me. It is definitely more clear and concise on what is supported.